### ISSUE TITLE
WCAG: How do we handle buttons in the table of our repeating groups components?

### ISSUE THREAD

**Issue by Febakke** [Altinn/app-frontend-react#252]
### Description

In the table listing the groups the user has added with the repeating groups we have an issue with the cell containing the edit button. Also we will add the delete button in this table as well, so we need to plan how we solve this for two buttons. 

Right now we have an empty header for the column that contains the button. We should look into if we need to add a hidden aria to describe what that column contains. We do no not need to show it as it is enough information in the button. We should also be careful about adding a header with the word "rediger" as we might end with screen readers saying words twice. 
[Relevant WCAG failure](https://www.w3.org/WAI/WCAG21/Techniques/failures/F91.html)

Because button gets focus when tabbing they will skip all informasjon in the table and jump directly too the button. I believe we should look into if we need to give some extra informasjon at this place so they have any informasjon about the rest of the row. What element am I editing if i press this button. 

Example using VoiceOver
![Skjermbilde 2022-06-24 kl  10 48 04](https://user-images.githubusercontent.com/33222679/175504828-0d6b0f05-130f-4fcb-a492-77e5614a6a63.png)
It do say something about the table and what column in the row you are at, but I think it would be easier to navigate if we presented some identifying informasjon about the row. For example something like: `rediger, knapp, Ola nordmann, kolonne 4 av 4`

### In scope

Research other solutions similar to this. How do they handle a button in a table?
Specify how this should be solved

**Comment by RonnyB71** [comment]
Solved
